### PR TITLE
Small update to DG and UG 

### PR DESCRIPTION
**Pull request by Wei-Xinn**
## Changes 
This PR only adds:
- findappt to UG 
- Onboarding to the glossary in DG.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -413,6 +413,23 @@ Example:
 
 ![Appointments Between Wireframe](images/FreeBetween.png)
 
+### Finding Appointment by Name or Person Name : `findappt`
+
+Finds appointments with name or Person name that contains the given keyword.
+
+Format: `findappt KEYWORD [MORE_KEYWORDS] [by/person]`
```
Comment (sharpstorm) [inline docs/UserGuide.md:420]: Small edit, this should be `[by/(person OR name)]`

File: docs/UserGuide.md [hunk 2]
```diff
@@ -413,6 +413,23 @@ Example:
...
+
+* The search is case-insensitive. e.g `Meeting` will match `meeting`
+* The order of the keywords does not matter. e.g. `Meeting Here` will match `Here Meeting`
+* Only full words will be matched e.g. `Meeting` will not match `Meetings`
+* Appointments matching at least one keyword will be returned (i.e. `OR` search).
+  e.g. `Meeting Here` will return `Meeting There`, `Meeting Now`
+* If `by/person` is included, the appointments will be matched based on the person name.
```
[the remark this answers is not in the document]
Comment (Wei-Xinn) [reply docs/UserGuide.md:427]: Added

File: docs/UserGuide.md (hunk 1 above)
Comment (Wei-Xinn) [reply docs/UserGuide.md:420]: Fixed

**Review by sharpstorm — APPROVED** [review]
LGTM.

**Pull request merged**